### PR TITLE
No nvram

### PR DESCRIPTION
**Pull request by jreidinger**
replaces https://github.com/yast/yast-bootloader/pull/616

- https://bugzilla.suse.com/show_bug.cgi?id=1157550

### CODE REVIEW

**Comment by jreidinger** [comment]
@watologo1 @hramrach I need your help here. Due to bug checker, we add to master workaround for arm at https://github.com/yast/yast-bootloader/pull/615 but I worry its logic collide. As now on arm we run with no-nvram --removable and also another grub2-install without it see https://github.com/yast/yast-bootloader/pull/617/files#diff-a2dd9a79782b903e9a1485d4c37037a7R44

So could you evaluate if we should remove this workaround or if logic is still valid?

**Comment by hramrach** [comment]
Yes, it will conflict. This needs to be rebased.

**Comment by watologo1** [comment]
Am Dienstag, 22. September 2020, 11:27:51 CEST schrieb Michal Suchánek:
> Yes, it will conflict. This needs to be rebased.

Sigh...
I hoped fixing up the minor coding style issue could help...

@Michal: Can you rebase this one, please. You know the code much better.

Looks like the Travis checks have been all addressed and calmed down.

I keep thumbs crossed we get this soon...

Thanks everybody,

    Thomas

**Comment by jreidinger** [comment]
well, it does not conflict as git conflict, just result is probably not what we want. I fix all travis issues, so please feel free to create pr against this branch and I will handle rest.

**Comment by hramrach** [comment]
Updated the original PR.

**Comment by jreidinger** [comment]
@hramrach thanks, I check it and adapt this branch to behave same.

**Comment by teclator** [comment]
From code point of view it looks good to me.

**Review by teclator — APPROVED** [review]
LGTM

File: src/autoyast-rnc/bootloader.rnc [hunk 1]
```diff
@@ -66,6 +66,7 @@ bl_global =
       element os_prober       { STRING_BOOL }? &
       element suse_btrfs      { STRING_BOOL }? &
       element secure_boot      { STRING_BOOL }? &
+      element update_nvram      { STRING_BOOL }? &
```
Comment (teclator) [inline src/autoyast-rnc/bootloader.rnc:69]: Do not forget to update the yast2-schema dependency.

https://github.com/yast/yast-schema/blob/master/package/yast2-schema.spec#L48

**Pull request merged**